packages/fxa-settings/src/lib/types.ts

```diff
@@ -0,0 +1,9 @@
+export interface QueryParams {
+  device_id?: string;
+  flow_id?: string;
+  flow_begin_time?: number;
+}
+
+export interface GenericObject {
+  [propName: string]: any;
+}
```

Comment:
I feel like "object" is already super overloaded.  Let's name it "hash"?

We could also add a type param so that it's possible narrow the value type, which in this case is string I think.

`Hash<T> {
  [key: string]: T;
}`

Something like that?